src/components/FeatureFlagModal/DevFlagsBox.tsx

```diff
@@ -0,0 +1,88 @@
+import { BaseVariant, FeatureFlag, featureFlagSettings } from 'featureFlags'
+import { useAtomValue } from 'jotai/utils'
+import { useMemo, useState } from 'react'
+import { GateResult, useGate } from 'statsig-react'
+import styled from 'styled-components'
+import { ThemedText } from 'theme/components'
+import { isDevelopmentEnv, isStagingEnv } from 'utils/env'
+
+const Box = styled.div`
+  position: fixed;
+  bottom: 20px;
+  left: 20px;
+  background-color: ${({ theme }) => theme.surface1};
+  padding: 10px;
+  border: 1px solid ${({ theme }) => theme.accent1};
+  z-index: 1000;
+  width: 300px;
+`
+
+export default function DevFlagsBox() {
+  const shouldShow = isDevelopmentEnv() || isStagingEnv()
+  const featureFlags = Object.entries(useAtomValue(featureFlagSettings))
```

Comment:
`Object.entries` will return a new object every render.. better to do that inside the memo below